### PR TITLE
python: fix argument parsing, missing css-beautify config option

### PR DESCRIPTION
**Pull request by DanielChabrowski**
# Description
- [x] Source branch in your fork has meaningful name (not `main`)

Please do notice that the default value of `keep_quiet` is different from js-beautify as there's no option to control it and I decided not to make it - python's part of the project doesn't seem very active anyway.

Fixes Issues: 
`-n` matching incorrect option. `("--preserve**-n**ewlines")` is not a tuple but a string.
```python3
>>> '-n' in ("--preserve-newlines")
True
>>> '-n' in ("--preserve-newlines",)
False
```

missing config option appearing if there are no changes:
```
'BeautifierOptions' object has no attribute 'keep_quiet'
```

keep_quiet didn't actually quiet `beautified FILE` message

# Before Merge Checklist 
These items can be completed after PR is created.

(Check any items that are not applicable (NA) for this PR)

- [ ] JavaScript implementation
- [x] Python implementation (NA if HTML beautifier)
- [ ] Added Tests to data file(s)
- [ ] Added command-line option(s) (NA if
- [ ] README.md documents new feature/option(s)

### CODE REVIEW

File: python/cssbeautifier/css/options.py [hunk 1]
```diff
@@ -57,3 +57,4 @@ def __init__(self, options=None):
             self._get_boolean("space_around_combinator")
             or space_around_selector_separator
         )
+        self.keep_quiet = True
```
Comment (bitwiseman) [inline python/cssbeautifier/css/options.py:60]: Shouldn't this default to `False`?

**Review by bitwiseman — CHANGES_REQUESTED** [review]
I guess there are no tests for these options? If there were, would they be broken? 😢

**Comment by DanielChabrowski** [comment]
> I guess there are no tests for these options? If there were, would they be broken? cry

I ran `make pytest` and nothing broke so I guess there aren't any for python implementation.

**Review by bitwiseman — APPROVED** [review]
I added a test that would fail before this change.  It's not perfect but it proves this issue is fixed and tries to limit it's return.

**Pull request merged**